### DISCUSSION TITLE
FEATURE: cursor interaction on  float multimonitor client;   multimonitor tiling client;

### DISCUSSION THREAD

**Discussion by fedoranvar** [hyprwm/Hyprland#4957] [General]
Good day to you!

I have two monitors vertically aligned.
I'm trying to emulate work of this monitors as one (virtual display) by:

## cursor interaction on float multimonitor  client

 stretching float client on two monitors
 
![image](https://github.com/hyprwm/Hyprland/assets/52028706/9d0f7332-7449-45ae-911d-7e9d2c1e30b3)

but cursor (mouse) clicks in client interface  (e.g. webbrowser) works only on one monitor

**Is it possible to make mouse cursor works on all monitors where client placed?**

## tiling client on multimonitor

**Is it possible to tile client on multimonitor?**

**Comment by vaxerski** [comment]
this is odd. are you on latest? it works just fine for me

**Reply by fedoranvar** [reply]
@vaxerski 
my bad. reason of odd behaviour was XWayland mode of Brave browser
changed mode to Wayland and now works fine.

and what about tiling client to multimonitors (as one big monitor)?
i've seen that X11 has that feature, to create one virtual display from multiple physical monitors

is it possible to make it with hyprland's power or it should be done on wlroots end?

**Reply by vaxerski** [reply]
it's governed by hyprland, but not a thing atm. there is an open issue about that.
